synapse/lib/stormlib/cache.py

```diff
@@ -0,0 +1,163 @@
+import asyncio
+
+import synapse.exc as s_exc
+
+import synapse.lib.ast as s_ast
+import synapse.lib.cache as s_cache
+import synapse.lib.stormctrl as s_stormctrl
+import synapse.lib.stormtypes as s_stormtypes
+
+CACHE_SIZE_MAX = 10_000
+CACHE_SIZE_DEFAULT = 10_000
+
+@s_stormtypes.registry.registerLib
+class LibCache(s_stormtypes.Lib):
+    '''
+    A Storm Library for interacting with Cache Objects.
+    '''
+    _storm_locals = (
+        {'name': 'fixed', 'desc': '''
+            Get a new Fixed Cache object.
+
+            When the callback Storm query is executed a special variable,
+            $cache_key, will be set. The query must contain a return statement.
+            If the query does not return an explicit value, $lib.null will set as the value.
```

Comment:
``If the query does not return a value when executed with the input, $lib.null will set as the value.``

Maybe something along these lines?

Reply:
sold